lib/helpers/UiBaseLocators.ts

```diff
@@ -60,14 +58,14 @@ export class UiBaseLocators {
     this.whereOperatorBtn = page.locator('umb-query-builder-filter').filter({hasText: 'where'}).getByLabel('Choose operator');
     this.whereConstrainValueTxt = page.locator('umb-query-builder-filter').filter({hasText: 'where'}).getByLabel('constrain value');
     this.orderByPropertyAliasBtn = page.locator('#sort-dropdown').getByLabel('Property alias');
-    this.acsendingBtn = page.locator('uui-button').filter({hasText: 'ascending'}).locator('#button');
+    this.ascendingBtn = page.locator('uui-button').filter({hasText: 'ascending'}).locator('#button');
     this.queryBuilderShowCode = page.locator('umb-code-block');
-    this.createBtn = page.getByLabel('Create...', {exact: true});
-    this.chooseDictionaryBtn = page.getByLabel('Choose', { exact: true });
-    this.newFolderMenu = page.getByLabel('New Folder...');
-    this.renameMenu = page.getByLabel('Rename...', {exact: true});
+    this.createThreeDotsBtn = page.getByLabel('Create...', {exact: true});
+    this.chooseThreeDotsBtn = page.getByLabel('Choose', {exact: true});
```

Comment:
This button doesn't have "..." so the name should be "chooseBtn" instead of "chooseThreeDotsBtn"